src/core/attributeformmodelbase.cpp

```diff
@@ -511,7 +511,10 @@ void AttributeFormModelBase::updateDefaultValues( int fieldIndex )
     if ( success && updatedValue != previousValue )
     {
       item->setData( defaultValue, AttributeFormModel::AttributeValue );
-      updateDefaultValues( fidx );
+      if ( !fromUpdatedDefaultValue )
```

Comment:
```
if( !updatedFields.contains( fidx ) )
{
  updatedFilds.append( fidx )
```

Let's just support cascading default values. Even if it's probably not the best idea, why not ...

Reply:
@m-kuhn , that's exactly what I was thinking last night when not feeling satisfied about the bool solution

Reply:
@m-kuhn , done.